package.json

```diff
@@ -66,7 +64,7 @@
     "google-auth-library": "8.1.0",
     "html-webpack-plugin": "4.5.2",
     "react-error-overlay": "6.0.9",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^2.1.3",
```

Comment:
Is this intentional? I can foresee github complaints about this as there are a ton of security issues with older transient dependencies in react-scripts